fix(schematic-sdk): support the multi tag in SDK's APIs (#1942)
AmadeusITGroup/otter: migration-guides/11.0.md

- `prepareOptions` method has been removed from `ApiClient`. `getRequestOptions` should be used instead.
- To support `JSON` specification format and to remove the reference to the outdated `Swagger` format name, the file containing the current specification used to generate the SDK will be stored in a file `open-api.yaml` or `open-api.json` instead of `swagger-api.yaml`.

> - To avoid interface name conflict in case of API with multiple tags (See issue [#855](https://github.com/AmadeusITGroup/otter/issues/855)), the API parameter object interface named with the pattern `<api-function-name>RequestData` will now be named with the `<api-function-name><api-name>RequestData`.

Reviewer: ```suggestion
- To avoid interface name conflicts in case of an API with multiple tags (See issue [#855](https://github.com/AmadeusITGroup/otter/issues/855)), the API parameter object interface named with the pattern `<api-function-name>RequestData` will now be named with the pattern `<api-function-name><api-name>RequestData`.
```

>   - The same renaming is applied to the `enum` types part of request parameters. It changes from `<api-function-name><parameter-name>Enum` to `<api-name><api-function-name><parameter-name>Enum`.

Reviewer: ```suggestion
  - The same renaming is applied to `enum` types in request parameters. It changes from `<api-function-name><parameter-name>Enum` to `<api-name><api-function-name><parameter-name>Enum`.
```
